docs/medair.md

```diff
@@ -0,0 +1,70 @@
+# nf-core/configs: Medair Configuration
+
+All nf-core pipelines have been successfully configured for use on the medair cluster at Clinical Genomics Gothenburg.
+
+To use, run the pipeline with `-profile medair`. This will download and launch the [`medair.config`](../conf/medair.config) which has been pre-configured with a setup suitable for the medair cluster. 
+It will enable Nextflow to manage the pipeline jobs via the `SGE` job scheduler.
+Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+## Download nf-core pipelines
+
+### Set-up
+
+First you need to load relevant softwares: Nextflow and nf-core tools. You can do it as follow:
+
+```bash
+## Load Nextflow
+module load nextflow/21.10.5.5658
+## Load nf-core tools
+module load miniconda
+source activate nf-core
+```
+
+### Download a pipeline
+
+We have started to download pipelines in the following location: `/apps/bio/repos/nf-core/`
+
+Use the `nf-core download --singularity-cache-only` command to start a download. It will open an interactive menu. Choose `singularity` for the software container image, and `none` for the compression type.
+
+### Storage of Singularity images
+
+When downloading a new nf-core pipeline for the first time (or a specific version of a pipeline), you can choose to store the Singularity image for future use. A central location for these images is: `/apps/bio/dependencies/nf-core/singularities`
+
+Cached Singularity images can be accessed by running (or adding to your `.bashrc`) the following: 
+
+```
+export NXF_SINGULARITY_CACHEDIR="/apps/bio/dependencies/nf-core/singularities"
```

Comment:
Could this somehow be be included in the profile? That way we wouldn't have to set it in the users shell

Reply:
Yes apparently it can  be included. I did it now. We should verify that it works though (I will write a comment if I test it).

Reply:
I tried to test it and in fact I do not know how to...
The cache directory is used e.g. for `nf-core download`, but that is not a Nextflow command and we do not give our cluster config there...
@sima-r , @pbiology , any idea?

Reply:
But it's also to tell the pipe where to look for singularities, no?

Reply:
Yes, that part is fine (running pipelines).
But when one downloads a new pipeline and wants to store the image, the cluster config is not involved. So I don't see how to export the variable in bashrc or such. Or am I missing something totally?